### DISCUSSION TITLE
Update method of code-server

### DISCUSSION THREAD

**Discussion by leaf2006** [coder/code-server#3506] [Q&A] [answered]
The code-server has been updated recently, and I want to know how to perform a hot update.

Installation method: npm

Hope someone can help me.
Thanks!

**Comment by nooobcoder** [comment]
What do you mean to say by a `hot update?` and **Installation method: npm**
Can you please be more descriptive about this?

**Comment by jsjoeio** [comment, marked as answer]
Assuming you've installed with npm, then you can run `npm install -g code-server@latest` which should install the latest version
